Paralogues.pm

```diff
@@ -0,0 +1,705 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ Paralogues
+
+=head1 SYNOPSIS
+
+ mv Paralogues.pm ~/.vep/Plugins
+
+ # Download Ensembl paralogue annotation (if not available in current directory)
+ # and fetch variants from Ensembl API (requires database access)
+ ./vep -i variations.vcf --plugin Paralogues
+
+ # Download Ensembl paralogue annotation (if not available in current directory)
+ # and fetch variants from custom VCF file
+ ./vep -i variations.vcf --plugin Paralogues,vcf=/path/to/file.vcf
+
+ # Fetch Ensembl variants in paralogue proteins using only the Ensembl API
+ # (requires database access)
+ ./vep -i variations.vcf --database --plugin Paralogues,mode=remote
+
+=head1 DESCRIPTION
+
+ A VEP plugin that fetches variants overlapping the genomic coordinates of amino
+ acids aligned between paralogue proteins. This is useful to predict
+ pathogenicity of variants in paralogue positions.
+
+ This plugin fetches variants overlapping the genomic coordinates of the
+ paralogue's affected aminoacid from one of two sources:
+ - Custom VCF based on vcf parameter
+ - Ensembl API (if vcf parameter is not defined)
+
+ This plugin automatically downloads paralogue annotation from Ensembl databases
+ if not available in the current directory (by default). Use argument `dir` to
+ change the directory of the paralogue annotation:
+   --plugin Paralogues
+   --plugin Paralogues,dir=/path/to/dir
+
+ It is also possible to point to a custom tabix-indexed TSV file by using
+ argument `paralogues`:
+   --plugin Paralogues,paralogues=file.tsv.gz
+   --plugin Paralogues,dir=/path/to/dir,paralogues=paralogues_file.tsv.gz
+   --plugin Paralogues,paralogues=/path/to/dir/paralogues_file.tsv.gz
+
+ The overlapping variants can be fetched from Ensembl API (by default) or a
+ custom tabix-indexed VCF file. You can point to a VCF file using argument `vcf`
+ and input a colon-delimited list of INFO fields in `cols` (by default, all
+ columns are returned):
+   --plugin Paralogues,vcf=/path/to/file.vcf.gz
+   --plugin Paralogues,vcf=/path/to/file.vcf.gz,cols=CLNSIG:CLNVI:GENEINFO
```

Comment:
Also supporting a VCF of ClinVar variants already mapped to paralogous locations, created during the release process, would be efficient